INSTALL.md

```diff
@@ -81,13 +81,31 @@ $ dpkg -i mycms*.deb
 #### Dependencies
 
 ```
-$ sudo apt install nsis
+$ sudo apt install mingw-w64 nsis
 ```
 
 #### Build
 
 ```
-$ ./configure --host=x86-w64-mingw32 --prefix=/ ...
-$ make install DESTDIR="$(pwd)/tmp"
-$ DESTDIR=tmp ./packaging/windows-nsis/build
+$ set MYCMS_PREFIX=/tmp/mycms-prefix
+```
+
+#### Openssh
+
+```
+$ ./Configure --prefix="${MYCMS_PREFIX}" --cross-compile-prefix=x86_64-w64-mingw32- mingw64
```

Comment:
@gilweis : this should be run in openssl sources :)

Reply:
please change from:
$ set MYCMS_PREFIX=/tmp/mycms-prefix
to
$ MYCMS_PREFIX=/tmp/mycms-prefix